sdk/identity/identity/src/credentials/managedIdentityCredential/arcMsi.ts

```diff
@@ -15,21 +21,22 @@ const logger = credentialLogger("ManagedIdentityCredential - ArcMSI");
 // Azure Arc MSI doesn't have a special expiresIn parser.
 const expiresInParser = undefined;
 
-function prepareRequestOptions(resource?: string): RequestPrepareOptions {
+function prepareRequestOptions(resource?: string): PipelineRequestOptions {
   const queryParameters: any = {
     resource,
     "api-version": azureArcAPIVersion
   };
 
+  const query = qs.stringify(queryParameters);
```

Comment:
same thought here with using `URLSearchParams`

Reply:
Will be done as part of #8841